.storybook/foundations/corner-rounding/corner-rounding.stories.js

```diff
@@ -0,0 +1,75 @@
+// Import the component markup template
+import { html } from "lit";
+import { Template } from './template';
+
+export default {
+	title: "Foundations/Corner Rounding",
```

Comment:
```suggestion
	title: "Foundations/Corner rounding",
```
Just a minor suggestion here to keep title case on the story name. This would be consistent with the capitalization on the Down state PR as well.